tests/unit/helper.py

```diff
@@ -1,14 +1,11 @@
 """Base classes and helpers for unit tests."""
-try:
-    from unittest import mock
-except ImportError:
-    import mock
 import github3
 import json
 import os.path
 import sys
 import pytest
 import unittest
+import unittest.mock
```

Comment:
I'd rather only use the backported module as it will backport features from 3.9+ to 3.6, 3.7, 3.8, etc.

Reply:
@sigmavirus24 I saw that there are pytest mixed with unittest in this project. If you decide to go full pytest in the future an alternative is using [pytest_mock](https://github.com/pytest-dev/pytest-mock/). But I can undo these changes if you prefer.

Reply:
Moving to pytest was always the goal

Reply:
In this case, I would leave these imports of mock as is in this PR and work on pytest tests in a new PR after merging. Deal?

Reply:
Sounds great